packages/grid/_modules_/grid/utils/utils.ts

```diff
@@ -22,18 +22,19 @@ export function getThemePaletteMode(palette: any): string {
 }
 
 export function isMuiV5(): boolean {
-  return 'alpha' in styles;
+  return !('fade' in styles);
 }
 
 export function muiStyleAlpha(color: string, value: number): string {
-  if (isMuiV5()) {
+  if (isMuiV5() || !(styles as any)?.fade) {
```

Comment:
Why not?

```suggestion
  if ((styles as any)?.alpha) {
```

x the other case

Reply:
I wanted to still utilize the `isMuiV5` helper but in those two cases, it doesn't make much sense.

We would still need to keep it tho, `isMuiV5` as it is used on other places as well.